horovod/spark/task/task_info.py

```diff
@@ -23,7 +24,7 @@ def __init__(self):
 
 
 def get_available_devices():
-    if 'gpu' not in _info.resources:
+    if 'gpu' not in _info.resources or os.getenv('USE_DEFAULT_GPU_INDEX') == '1':
```

Comment:
If this is specific to horovod, i suggest to append it with "HOROVOD_" like "HOROVOD_USE_DEFAULT_GPU_INDEX" to keep it consistent with other horovod envs.

Reply:
Env name updated. Thanks.